pkg/services/audit/auditor/pop.go

```diff
@@ -1,15 +1,14 @@
 package auditor
 
 import (
-	"github.com/nspcc-dev/neofs-sdk-go/client"
 	"github.com/nspcc-dev/neofs-sdk-go/netmap"
 	oidSDK "github.com/nspcc-dev/neofs-sdk-go/object/id"
 	"go.uber.org/zap"
 )
 
 const (
 	hashRangeNumber    = 4
-	minGamePayloadSize = hashRangeNumber * client.TZSize
+	minGamePayloadSize = hashRangeNumber * 64
```

Comment:
Could you make a named constant or at least add a comment about `64`?

Reply:
Used recently defined `const tz.Size`.